src/js/app.js

```diff
@@ -1,5 +1,59 @@
 import $ from "jquery";
+import 'slick-carousel';
 
 $(function() {
   console.log('環境構築完了');
 })
+
+$(function() {
+  //ハンバーガーボタン⇨メニュースクロール
+  $('.l-header__hamburger').on('click', function() {
+    $('.l-header__hamburger').toggleClass('open-btn');
+    $('.header-nav').toggleClass('open-menu');
+    
+    if ($("body").css("overflow") === "hidden") {
```

Comment:
[nits]

シングルクォーテーションで統一させましょう

```suggestion
    if ($('body').css('overflow') === 'hidden') {
```